keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -161,7 +181,7 @@ We also plan to test this with different version Skews.
 
 ### Graduation Criteria
 
-This KEP will be treated as a new feature, and will be introduced with a new feature gate, `NonGracefulFailover`.
+This KEP will be treated as a new feature, and will be introduced with a new feature gate, `NodeOutOfServiceVolumeDetach`.
```

Comment:
Do we want to tie the feature name to VolumeDetach? Or is force deletion the higher level behavior that this provides?

Reply:
Naming is the hardest thing:).  I think we had "force" in the name earlier on, but got suggestion to change to this current name from some review comments.